modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying the architecture of an Agent-based installer cluster
+
+After installing an {product-title} cluster using the Agent-based Installer, you can check the architecture of the cluster.
```

Comment:
```suggestion
After installing an {product-title} cluster using the Agent-based Installer, you can check the architecture of the cluster. This procedure is optional.
```
Perhaps add a phrase about why a user might want to do this optional step?

Reply:
Updated as per the suggestion. 

"Before installing an OpenShift Container Platform cluster using the Agent-based Installer, you can verify the supported architecture on which you can install the cluster."

This procedure helps to get the supported arch that users can use to install the cluster.